ethportal-api/src/types/cli.rs

```diff
@@ -255,6 +263,20 @@ fn check_private_key_length(private_key: &str) -> Result<H256, String> {
     ))
 }
 
+fn check_trusted_block_root(trusted_root: &str) -> Result<String, String> {
```

Comment:
This seems like an obvious candidate to be pulled out into `trin-utils` and consolidated w/ [this implementation](https://github.com/ethereum/trin/blob/092427a4f0babf6a8bc3423de84a89b4f244d0cd/ethportal-api/src/types/cli.rs#L248-L256)

Reply:
Not sure, I don't want to add `trin-utils` as a dependency of `ethportal-api` because ethportal-api should exist independently from trin.